plugins/modules/device.py

```diff
@@ -97,6 +97,12 @@
     required: false
     type: str
     version_added: "3.0.0"
+  software_version:
+    description:
+      - The software version of the device
+    required: false
+    type: str
+    version_added: "3.0.0"
```

Comment:
```suggestion
    version_added: "5.12.0"
```

This needs to be the next minor version.